libopenage/util/fslike/directory.cpp

```diff
@@ -90,11 +91,11 @@ bool Directory::is_file(const Path::parts_t &parts) {
 bool Directory::is_dir(const Path::parts_t &parts) {
 	auto stat_result = this->do_stat(parts);
 
-	// test for regular file
-	if (std::get<1>(stat_result) == 0 and
-	    S_ISDIR(std::get<0>(stat_result).st_mode)) {
-		return true;
-	}
+	// test for existing directory
+    if (stat_result and
+        sf::is_directory(stat_result)) {
+        return true;
+    }
```

Comment:
I think the indentation should be changed to tabs to remain consistent with the other parts of this project.

Reply:
Also on line 96 I think sf:: is a typo

Reply:
Yes, thank you! I thought it's be more readable since the method names get quite lenghty in places.

And correct, line 96 was a typo, the code was written before I had a way to test this properly, will be fixed next commit ☺️

Reply:
@sl1g18 No problem, we will try to provide a `.clang-format` file soon so the formatting shouldn't be any major problem to keep track of. You can take a look at it [here](https://github.com/SFTtech/openage/blob/564356ec6b586a4fd88fe7ad3458faa6339dbe53/.clang-format).